docker_files_test.go

```diff
@@ -66,7 +66,7 @@ func TestCopyFileToRunningContainer(t *testing.T) {
 
 	ctr, err := testcontainers.GenericContainer(ctx, testcontainers.GenericContainerRequest{
 		ContainerRequest: testcontainers.ContainerRequest{
-			Image: "docker.io/bash:5.2.26",
+			Image: "bash:5.2.26",
```

Comment:
question: others don't have a version, why is this one special?

Reply:
No special at all, I wonder why it was not included for all the occurrences. I think we can change that in this PR